pyvista/plotting/plotting.py

```diff
@@ -485,9 +486,13 @@ def export_gltf(self, filename, inline_data=True, rotate_scene=True, save_normal
                                 continue
                             dataset = mapper.GetInputAsDataSet()
                             if 'Normals' in dataset.point_data:
-                                # ensure normals are active
-                                normals = dataset.point_data['Normals']
-                                dataset.point_data.active_normals = normals.copy()
+                                # By default VTK uses the 'Normals' point data for normals
+                                # but gLTF uses NORMAL.
+                                point_data = dataset.GetPointData()
+                                array = point_data.GetArray('Normals')
+                                array.SetName('NORMAL')
+                                renamed_arrays.append(array)
```

Comment:
Note: I'm using the underlying VTK API here as there's an odd memory leak when using `DataSetAttributes`.